src/material/datepicker/datepicker.md

```diff
@@ -1,6 +1,24 @@
 The datepicker allows users to enter a date either through text input, or by choosing a date from
 the calendar. It is made up of several components and directives that work together.
 
+To start using the datepicker you will need to import the datepicker module and set the datepicker adapter to use:
+
+```ts
+import { NgModule } from '@angular/core';
```

Comment:
We don't put spaces around the braces: `import {NgModule} from '@angular/core';`